python/raft-dask/pyproject.toml

```diff
@@ -34,9 +34,10 @@ authors = [
 license = { text = "Apache 2.0" }
 requires-python = ">=3.8"
 dependencies = [
+    "dask-core==2023.3.2",
```

Comment:
Is this dask-core dependency required? I don't see it up on pypi - and we're seeing some CI failures like 
`No matching distribution found for dask-core==2023.3.2`

Reply:
Fixed it, you are right it's only conda specific.